### PR TITLE
LC-436: create output directory

### PR DESCRIPTION
**Pull request by akul-sethi**
patch for old LC-436 so that output directory is created.

### CODE REVIEW

File: lucille-core/src/test/java/com/kmwllc/lucille/core/FileToFileIngestIT.java [hunk 1]
```diff
@@ -15,6 +15,7 @@ public void fullIT() throws Exception {
     File f = null;
     try {
       Config config = ConfigFactory.parseFile(new File("src/test/resources/FileToFileIngestIT/file-to-file-example.conf"));
+      new File("output/").mkdir();
```
Comment (rseitz) [inline lucille-core/src/test/java/com/kmwllc/lucille/core/FileToFileIngestIT.java:18]: pls. call the directory `tempIntegrationTestOutput`
pls. delete the directory after the test

**Review by rseitz — APPROVED** [review]

File: lucille-core/src/test/java/com/kmwllc/lucille/core/FileToFileIngestIT.java [hunk 2]
```diff
@@ -13,9 +13,12 @@ public class FileToFileIngestIT {
   @Test
   public void fullIT() throws Exception {
     File f = null;
+    File directory = null;
     try {
       Config config = ConfigFactory.parseFile(new File("src/test/resources/FileToFileIngestIT/file-to-file-example.conf"));
-      f = new File("output/dest.csv");
+      directory = new File("tempIntegrationTestOutput/");
+      directory.mkdir();
+      f = new File("tempIntegrationTestOutput/dest.csv");
```
Comment (kiratraynor) [inline lucille-core/src/test/java/com/kmwllc/lucille/core/FileToFileIngestIT.java:21]: I think the `file-to-file-example.conf` will need to be updated tho this new tempIntegrationTestOutput dir as well

File: lucille-core/src/test/java/com/kmwllc/lucille/core/FileToFileIngestIT.java [hunk 3]
```diff
@@ -29,6 +32,7 @@ public void fullIT() throws Exception {
       assertEquals("\"Salad\",\"USA\",\"10\"", lines.get(6));
       assertEquals("\"Wine\",\"France\",\"12\"", lines.get(7));
     } finally {
+      directory.delete();
       f.delete();
```
Comment (kiratraynor) [inline lucille-core/src/test/java/com/kmwllc/lucille/core/FileToFileIngestIT.java:36]: The directory and file delete lines would need to be swapped if we want to delete the directory too

**Comment by rseitz** [comment]
Agree with Kira's comment
We'll need a followup fix

**Pull request merged**